sfdx-source/apex-common/main/classes/fflib_SObjects.cls

```diff
@@ -161,11 +161,13 @@ public virtual class fflib_SObjects
 	 * @return Return a set with all the Id values of the given field
 	 */
 	@TestVisible
-	protected Set<Id> getIdFieldValues(Schema.SObjectField field)
+	protected virtual Set<Id> getIdFieldValues(Schema.SObjectField field)
 	{
 		Set<Id> result = new Set<Id>();
 		for (SObject record : getRecords())
 		{
+			if (record.get(field) == null) continue;
```

Comment:
@wimvelzeboer please add braces for readability.  Otherwise, looks good to me.

Reply:
@ImJohnMDaniel you are indeed right, I changed it!